build.gradle

```diff
@@ -375,12 +377,30 @@ task github(dependsOn: remapMavenJar) {
 		releaseBuilder.name("[$Globals.mcVersion] Fabric API $Globals.baseVersion")
 		releaseBuilder.body("A changelog can be found at https://github.com/FabricMC/fabric/commits")
 		releaseBuilder.commitish(getBranch())
+		releaseBuilder.prerelease(Globals.preRelease)
 
 		def ghRelease = releaseBuilder.create()
 		ghRelease.uploadAsset(file("${project.buildDir}/libs/${archivesBaseName}-${version}.jar"), "application/java-archive");
 	}
 }
 
+task modrinth(type: com.modrinth.minotaur.TaskModrinthUpload, dependsOn: remapMavenJar) {
+	onlyIf {
+		ENV.MODRINTH_TOKEN
+	}
+
+	token = ENV.MODRINTH_TOKEN
```

Comment:
This will be fine for now, I do need to go make minotaur use properties instead of public fields - it shouldn't break if I believe when I do that.

Reply:
What's wrong with this using env?

Reply:
There isn't an issue with the token in the ENV - but rather it's a nasty bit of the gradle plugin and how it exposes it's api to consumers. Not a blocking issue but I plan to neaten up the plugin upstream.

Not related to FAPI here - but Minotaur stuff inside of the plugin.

Typically you will want to use a getter + setter with an `@Input` annotation in gradle, minotaur uses public mutable fields atm which is a bit of code smell. In groovy the `field = value` is essentially adapted into a `get/set` like how kotlin handles var types.

Reply:
if you update the plugin dependency gradle will recompile the buildscript, so this is a non-issue on fabric's end

Reply:
Yes - and I believe I can do the upstream changes fine in the future without the buildscript breaking